src/wp-admin/site-editor.php

```diff
@@ -56,7 +56,13 @@ static function ( $classes ) {
 	$indexed_template_types[] = $template_type;
 }
 
-$block_editor_context = new WP_Block_Editor_Context( array( 'name' => 'core/edit-site' ) );
+$context_settings = array( 'name' => 'core/edit-site' );
+
+if ( ! empty( $_GET['postId'] ) ) {
```

Comment:
Let's verify that `postId` is an int value.

Reply:
Done